lib/theme_template/example/index.html

```diff
@@ -5,7 +5,7 @@
 Lorem ipsum dolor sit amet, quo id prima corrumpit pertinacia, id ius dolor dolores, an veri pertinax explicari mea. Agam solum et qui, his id ludus graeco adipiscing. Duis theophrastus nam in, at his vidisse atomorum. Tantas gloriatur scripserit ne eos. Est wisi tempor habemus at, ei graeco dissentiet eos. Ne usu aliquip sanctus conceptam, te vis ignota animal, modus latine contentiones ius te.
```

Comment:
I'm just thinking that we're initializing it with a theme, but when you view the site, it'll be unthemed out of the box.